packages/ocaml-basics/ocaml-basics.0.1.0/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "1.2"
+maintainer: "Antoine Luciani <aluciani@advanced-schema.com>"
+authors: "Advanced Schema"
+homepage: "http://github.com/advanced-schema/ocaml-basics"
+bug-reports: "http://github.com/advanced-schema/ocaml-basics/issues"
+license: "MIT"
+dev-repo: "https://github.com/advanced-schema/ocaml-basics.git"
+build: [
+  ["oasis" "setup"]
+  ["./configure" "--prefix=%{prefix}%"]
+  [make]
+]
+install: [make "install"]
+remove: ["ocamlfind" "remove" "ocaml-basics"]
+depends: [
+  "ocamlfind" {build}
+  "oasis" {>= "0.4"}
+  "result" {>= "1.2"}
+  "ppx_sexp_conv" {>= "0.9"}
+  "sexplib" {>= "0.9"}
```

Comment:
this one's a bit non-obvious -- perhaps we should put a note into `opam-repository/CHANGES` @diml? explaining the cunning reasoning behind the `v`? :-)